src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -49,6 +49,17 @@ class LedgerRPC_test : public beast::unit_test::suite
             BEAST_EXPECT(jv[jss::error_message] == msg);
     };
 
+    // Corrupt a valid address by replacing the 10th character with '!'.
+    // '!' is not part of the ripple alphabet.
+    std::string
+    makeBadAddress (std::string const& good)
```

Comment:
We could take the parameter by value, move it into `ret` (to take advantage of RVO) and save some copies. No change needed, as we don't really care about the run time cost of this function.